edweiss-firebase/functions/src/functions/lectures/likeQuestion.ts

```diff
@@ -21,7 +21,7 @@ export const updateQuestion = onAuthentifiedCall(LectureDisplay.Functions.likeQu
         username: question.username
     };
     try {
-        console.log(updateQuestion);
+        console.log(likeQuestion);
```

Comment:
you can remove this console.log, or change it to updatedQuestion if you want, since this was also a typo of my part

Reply:
Roger that, I didn't wanted to alter it as you might was needing it for debugging purposes. 
Now that I have your go, I do it right away.